examples/Makefile

```diff
@@ -520,7 +520,8 @@ SHADERS = \
     shaders/shaders_spotlight \
     shaders/shaders_hot_reloading \
     shaders/shaders_mesh_instancing \
-    shaders/shaders_multi_sample2d
+    shaders/shaders_multi_sample2d \
+    shaders/write_depth
```

Comment:
I wrote write_depth instead of shaders_write_depth.
Time to figure out how to fix this